runtime/lua/vim/ui/float.lua

```diff
@@ -0,0 +1,214 @@
+local Border = require('vim.ui.border')
+local tbl = require('vim.ui.ui_table')
```

Comment:
`tbl` can probably just be moved to `ui._utils` or something, and then you can do `tbl_apply_defaults` or something. I think this makes it seem like a much larger module than anticipated.

I would not consider `_utils` something that we would necessarily expose for awhile.